tests/testthat/test-2merge.R

```diff
@@ -2,47 +2,47 @@ context("merge")
 test_that("merge", {
     # .check_f
     expect_error(mia:::.norm_f(),
-                 'argument "f" is missing')
+                 'argument "group" is missing')
     expect_error(mia:::.norm_f(6),
-                 'argument "f" is missing')
+                 'argument "group" is missing')
     expect_error(mia:::.norm_f(6,5),
-                 "'f' must be a factor or character vector")
-    f <- factor(c(rep("a",3),rep("b",3)))
-    expect_true(is.factor(mia:::.norm_f(6,f)))
+                 "'group' must be a factor or character vector")
+    group <- factor(c(rep("a",3),rep("b",3)))
+    expect_true(is.factor(mia:::.norm_f(6,group)))
     # .check_archetype
     expect_error(mia:::.norm_archetype(),
                  'argument "archetype" is missing')
-    expect_error(mia:::.norm_archetype(f),
+    expect_error(mia:::.norm_archetype(group),
                  'argument "archetype" is missing')
-    expect_error(mia:::.norm_archetype(f),
+    expect_error(mia:::.norm_archetype(group),
                  'argument "archetype" is missing')
-    expect_equal(mia:::.norm_archetype(f, 1),c(1,1))
-    expect_equal(mia:::.norm_archetype(f, c(1,2)),c(1,2))
-    expect_error(mia:::.norm_archetype(f, c(1,2,3)),
+    expect_equal(mia:::.norm_archetype(group, 1),c(1,1))
+    expect_equal(mia:::.norm_archetype(group, c(1,2)),c(1,2))
+    expect_error(mia:::.norm_archetype(group, c(1,2,3)),
                  "length of 'archetype' must have the same length as levels")
-    expect_error(mia:::.norm_archetype(f, c(5)),
-                 "'archetype' out of bounds for some levels of 'f'")
+    expect_error(mia:::.norm_archetype(group, c(5)),
+                 "'archetype' out of bounds for some levels of 'group'")
     # .norm_archetype
     expect_error(mia:::.norm_archetype(),
                  'argument "archetype" is missing')
-    expect_error(mia:::.norm_archetype(f),
+    expect_error(mia:::.norm_archetype(group),
                  'argument "archetype" is missing')
-    actual <- mia:::.norm_archetype(f, c(1,2))
+    actual <- mia:::.norm_archetype(group, c(1,2))
     expect_equal(actual, c(1,2))
-    actual <- mia:::.norm_archetype(f, c(1))
+    actual <- mia:::.norm_archetype(group, c(1))
     expect_equal(actual, c(1,1))
 
```

Comment:
Most of these changes are not necessary as f is only variable. Avoid unnecessary changes